MovingBlocks/Terasology
# Terra bullet (gdx native)

This implementation uses a version of libgdx bullet extension to work directly with Termath along with a voxel implementation. there are some out standing features that need to be address and fixed before  this would be considered workable but this implementation works against the current development branch of Terasology.

https://github.com/MovingBlocks/Terasology/pull/3121
https://github.com/MovingBlocks/Terasology/pull/3042

https://github.com/pollend/TeraBullet-2

# How to Test

~~You will have to include the modified libgdx library into lib and install the maven libraries from there (mvn install). At the moment the way to load the native library is to just copy the native-build into the native folder in the root of the project. running the ant script in TeraMath should do everything for you in terms of building the library.~~

The bullet builds are in artifactory so there is no need to copy anything into lib.
